docs/UserGuide.adoc

```diff
@@ -400,29 +399,51 @@ To match all the patients in ORGANice, do the following:
 image::Patient Matches Displayed.png[width=790]
 +
 
+Following this, you can sort the matches based on different categories to deduce which patient to prioritize.
++
+Refer to refer to <<Sorting matches: `sort`>> for more information.
 
 === Sorting matches: `sort`
-Users can specify how they want the patient-donor pairs to be sorted. +
-Sorting is done by passing in optional parameters to the `match` command. +
+This section specifies how a list of matches can be sorted to determine which patient to prioritize for organ transplant.
```

Comment:
Better tone to sound as if you are talking with the user? Such as, `After getting the list of potential donors, you can ........`

Reply:
Amended.